server/src/main/java/io/deephaven/server/barrage/BarrageMessageProducer.java

```diff
@@ -996,6 +1005,14 @@ private void schedulePropagation() {
     private class UpdatePropagationJob implements Runnable {
         private final ReentrantLock runLock = new ReentrantLock();
         private final AtomicBoolean needsRun = new AtomicBoolean();
+        private final ExecutionContext executionContext;
+
+        UpdatePropagationJob(final OperationInitializer operationInitializer) {
+            this.executionContext = ExecutionContext.newBuilder()
+                    .setOperationInitializer(operationInitializer)
```

Comment:
My inclination was not to use dagger injection, but rather to use the `FJPOI.fromCommonPool()`.

Reply:
Let's defer the dagger-injected extra thread pool for now, use the common pool in the initial merge, and then decide whether to re-introduce the egress pool once we've had some bake time.